Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity_test.py

```diff
@@ -705,3 +706,68 @@ def test_get_message_user(mocker, user, expected_result):
                         return_value={'value': [{"id": "test user id"}]})
     message_user = get_message_user(client_mocker, user)
     assert message_user == expected_result
+
+
+def test_advanced_hunting_command(mocker):
+    response = load_json('./test_data/advanced_hunting_response.json')
+    mocker.patch.object(client_mocker, "advanced_hunting_request", return_value=response)
+    args = {'query': 'AlertInfo', 'limit': 2, 'timeout': 50}
+
+    results = advanced_hunting_command(client_mocker, args)
+
+    expected_results = load_json('./test_data/advanced_hunting_results.json')
+    assert results.outputs_prefix == expected_results['outputs_prefix']
+    assert results.outputs_key_field == expected_results['outputs_key_field']
+    assert results.outputs == expected_results['outputs']
+    assert results.readable_output == expected_results['readable_output']
+
+    mocker.patch.object(demisto, 'params', return_value={'microsoft_365_defender_context': True})
+    results = advanced_hunting_command(client_mocker, args)
+
+    expected_results = load_json('./test_data/advanced_hunting_results_365_defenfer.json')
+    assert results[1].outputs_prefix == expected_results['outputs_prefix']
+    assert results[1].outputs_key_field == expected_results['outputs_key_field']
+    assert results[1].outputs == expected_results['outputs']
+    assert results[1].readable_output == expected_results['readable_output']
+
+
+def test_get_list_security_incident_command_1(mocker):
```

Comment:
be more informative with the name of the test.

Reply:
I have updated